index.html

```diff
@@ -0,0 +1,129 @@
+            <button type="button" class="join-button" id="join">Join Viable Group GS 2022</button>
+            <h3 class="see-program">SEE THE WHOLE PROGRAM</h3>
+        </section>
+
+        <section class="speakers">
+            <h1 class="header-speakers">Featured Speakers</h1>
+            <hr class="line">
+
+            <div class="speakers-container">
+                <!--Dynamically Generated-->
+            </div>
+        </section>
+
+        <section class="about" id="about">
+            <h1 class="headline-hello">Achievers world</h1>
+            <h2 class="about-summit"> VIBRANT AFRICANS' YOUTH SUMMIT 2022</h2>
+            <p class="about-para">A global network of professional Software Developers 
+                aimed at sharing opportunities and creating a positive change 
+                with people from over 80 countries is taking place in Nigeria.
+            </p>
+            <p class="about-contact">Please contact us via Email for any further 
+                questions about VG Youth Summit 2022!<br><span class="about-mail">vgsummit2022@vggroup.org</span>
+            </p>
+        </section>
+
+        <section class="summit-logo">
+            <h1 class="summit-logo-header">Viable Group Youth Summit 2022 Logo</h1>
+            <hr class="line">
+            <p class="summit-logo-para">The logo of VG Youth Summit 2022 was decided through the logo compettition from 8, June to 7, July, 2021.</p>
+            <p class="summit-logo-para2">After online public voting and selection Committees Judging process, 
+                the logo from Hammed Adisa won the prize.<br> After consultation with Mr. Adisa we optimized 
+                the prize-winning-logo for this year's summit as following:
+            </p>
+            <div class="vg-logo">
+                <img src="images/vg-logo-about.png" alt="vg-logo">
+            </div>
+        </section>
+
+        <section class="past-summit" id="news">
+            <h1 class="past-summit-header">See the past VG Youth Summits</h1>
+            <hr class="line">
+            <p class="past-summit-para">Take a look at the last two VG Youth Summits
+                 which took place in Abuja and Port Harcourt.
+            </p>
+            <div class="past-summits-images">
+                <img src="images/pastSummit.PNG" alt="past summit">
+                <img src="images/pastSummit2.PNG" alt="past summit">
+            </div>
+        </section>
+
+        <section class="partner">
+            <h1 class="main-program">Partner</h1>
+            <hr class="line">
+            <div class="partner-list">
+                <img src="images/logo-jekaplay.png" alt="jekaplay">
+                <img src="images/logo-afrotainment.PNG" alt="afrotainment">
+            </div>
+        </section>
+
+        <footer class="footer-section" id="sponsor">
+            <h5 class="copyright">2022 Viable Group Conference Copy Rights Reserved.</h5>
+        </footer>
```

Comment:
- [ ] Please make sure that footer contains a logo aligned to the left and text aligned to the right. Currently I can only see centred text: 
<img width="408" alt="Screenshot 2022-04-13 at 20 21 30" src="https://user-images.githubusercontent.com/77038610/163225842-7ee8f21b-3995-4b1f-a58c-e6d81a17ec31.png">